client/src/components/ProductDetails/StyleSelector.jsx

```diff
@@ -1,41 +1,30 @@
+/* eslint-disable react/jsx-key */
 /* eslint-disable react/prop-types */
 import React from 'react';
 import './prodStyles.css';
 
-//can get rid of - simply get from
-function StyleSelector ({productStylesArray, currentStyleID, currentProductID, changeCurrentStyle}) {
-  // const getCurrentProductObject = (targetId, allProducts) => {
-  //   return allProducts.filter((product) => {
-  //     return product.product_id === targetId;
-  //   });
-  // };
+function StyleSelector ({productStylesArray, currentStyleID, changeCurrentStyle}) {
 
-  // const currentProductObject = getCurrentProductObject(currentProductID, productStylesArray);
 
   const getCurrentStyleObject = (targetStyleId, stylesArray) => {
     return stylesArray.filter((style) => {
       return style.style_id === targetStyleId;
     });
   };
 
-  // const stylesArray = currentProductObject[0]['results'];
-
   const currentStyleObject = getCurrentStyleObject(currentStyleID, productStylesArray);
 
-
   return (
-    <div clasName="container">
+    <div className="flex-container wrap">
       <h2>{currentStyleObject[0]['name']}</h2>
       <h4>{`$${currentStyleObject[0]['original_price']}`}</h4>
 
       {productStylesArray.map((style) =>
```

Comment:
if you change this to style, index and then pass in a key prop such as `$some-classname-{index}`, I think you could resolve the react/jsx-key error

Reply:
And then you could delete the first line of code in this file.